File: Conexion base de datos/PHP/mongodb.php
<?php
try {
    // Conexión básica a MongoDB
    $client = new MongoDB\Driver\Manager("mongodb://localhost:27017");
    
    // Comando para verificar la conexión (ping)
    $command = new MongoDB\Driver\Command(['ping' => 1]);
    $client->executeCommand('admin', $command);
    
    echo "✅ Conexión exitosa a MongoDB";
    
} catch (MongoDB\Driver\Exception\Exception $e) {
    echo "❌ Error de conexión: " . $e->getMessage();
}
?>

File: Conexion base de datos/GO/MongoDB.go
package main

import (
	"context"
	"fmt"
	"log"
	"time"

	"go.mongodb.org/mongo-driver/mongo"
	"go.mongodb.org/mongo-driver/mongo/options"
)

func main() {
	// Configuración de conexión
	clientOptions := options.Client().ApplyURI("mongodb://localhost:27017")
	
	// Conectar con timeout de 10 segundos
	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
	defer cancel()
	
	client, err := mongo.Connect(ctx, clientOptions)
	if err != nil {
		log.Fatal(err)
	}
	
	// Verificar conexión
	err = client.Ping(ctx, nil)
	if err != nil {
		log.Fatal(err)
	}
	
	fmt.Println("✅ Conexión exitosa a MongoDB")
	
	// Cerrar conexión cuando termine
	defer func() {
		if err = client.Disconnect(ctx); err != nil {
			log.Fatal(err)
		}
	}()
}